### PR TITLE
fix #18: fix README

### LINKED ISSUE #18: Update Documentation
The Readme file should be updated.
Documentation about the commands should be updated as well and well formatted.

### PR DESCRIPTION
**Pull request by chermehdi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,20 +7,34 @@ and help execute tests locally via their favorite programming language.
 
 ## Installation
 
-- The installation requires that you have `go` installed, and you just type `go get github.com/chermehdi/go-egor`
-and you will have the go-egor command available to you.
+- There is actually two ways for installing egor.
```
Comment (Eroui) [inline README.md]: There are two ways...

File: README.md [hunk 2]
```diff
@@ -7,20 +7,34 @@ and help execute tests locally via their favorite programming language.
...
+
+### Download compiled binary
+- You can download the binary that corresponds to your operating system and add it to your `PATH` variable, you can find the binaries in the [releases](https://github.com/chermehdi/go-egor/releases) page.
+- For people running Mac OSX Catalina, this won't work and you will be forced to go with the Build from source solution.
+
+### Build from source
+- You can clone the repository, and have go installed in your machine
+- Navigate to the directory of the cloned project and run `go build` to build the project, and `go install` to install it to your local machine, the binary can then be found in `$GOOPATH/bin/egor`
```
Comment (Eroui) [inline README.md]: $GOPATH

File: README.md [hunk 3]
```diff
@@ -7,20 +7,34 @@ and help execute tests locally via their favorite programming language.
...
 - The current supported command list is outlined here, and you can find out more details in the docs page.
-    - `go-egor parse`: Starts listening for competitive companion chrome plugin to parse the task
-    - `go-egor test`: Runs the tests of the current task and outputs the results
-    - `go-egor copy`: Copies the current task to the clipboard.
-    - `go-egor batch`: Tests the main solution for this task against another solution (probably written in another language than the main one)
+    - `egor parse`: Starts listening for competitive companion chrome plugin to parse the task
+    - `egor test`: Runs the tests of the current task and outputs the results
+    - `egor config`: Read/Change global configuration parameters.
+    - `egor testcase`: Add a custom test case to this egor task.
+    - `egor showcases`: list meta data abou the tests in the current task
```
Comment (Eroui) [inline README.md]: about

File: README.md (hunk 1 above)
Comment (chermehdi) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (chermehdi) [reply README.md]: fixed

**Pull request merged**